packages/klevu-ui/src/components/klevu-chat-messages/klevu-chat-messages.tsx

```diff
@@ -71,6 +80,35 @@ export class KlevuChatMessages {
 
   feedbackReasons = ["Irrelevant", "Incorrect", "Offensive", "Other"]
 
+  @State() lastMessageDisplayedText: string = '';
+  @State() typeWriterEnds: boolean = true;
+  
+  @Watch('messages')
+  watchPropHandler(newValue: any, oldValue: MoiMessages) {
+    const lastItem = newValue.at(-1) || null;
```

Comment:
at() is a new feature and isnt supported prior to 2022 browser versions, lets use standard array 
`newValue[newValue.length - 1]`

Reply:
changed to newValue[newValue.length - 1]